tendrl/commons/utils/ssh/authorize_key.py

```diff
@@ -65,6 +65,7 @@ def run(self):
                                             str(e.message))}
                 )
             )
+            err = str(e.message)
```

Comment:
please file a different issue and add another line in commit msg "tendrl-bug-id: Tendrl/commons#$thisbug"